$NetBSD: patch-configure.ac,v 1.13 2024/04/15 10:25:57 micha Exp $

Use the same path for all platforms in pkgsrc.
Fix nonportable shell syntax.

--- configure.ac.orig	2024-02-10 22:00:40.000000000 +0000
+++ configure.ac
@@ -92,8 +92,7 @@ if test -z "${LIBPOSTFIX++}"; then
   case "${host_os}" in
     *linux* )
       case "${host_cpu}" in
-        aarch64 | powerpc64 | powerpc64le | s390x | x86_64 | sparc64 | mips64* | riscv64 | e2k )
-          LIBPOSTFIX="64"
+        aarch64* | powerpc64 | powerpc64le | s390x | x86_64 | sparc64 | mips64* | riscv64 | e2k )
           INTGOSIZE=64
           ;;
       esac
@@ -101,7 +100,6 @@ if test -z "${LIBPOSTFIX++}"; then
     *solaris* )
       case "${host_cpu}" in
         x86_64 | sparc64 )
-          LIBPOSTFIX="/64"
           INTGOSIZE=64
           ;;
       esac
@@ -1760,7 +1758,10 @@ AM_CONDITIONAL([WITH_QT], [test "$use_qt
 dnl -----------------------------------
 dnl INCLUDES and LIBS for QUARTZ
 
-if test "${host_os::6}" = "darwin"; then
+dnl Not suitable for a POSIX shell
+dnl if test "${host_os::6}" = "darwin"; then
+case ${host_os} in
+darwin*)
   AC_ARG_WITH(quartz,
     [AS_HELP_STRING([--with-quartz=yes],[Quartz framework (Mac OS X)])],
     [],[with_quartz=yes])
@@ -1777,10 +1778,11 @@ if test "${host_os::6}" = "darwin"; then
       use_quartz="No (missing ApplicationServices.framework)"
     fi
   fi
-else
+  ;;
+*)
   with_quartz=no
   use_quartz="No (disabled by default - Mac only)"
-fi
+esac
 AM_CONDITIONAL(WITH_QUARTZ, [test "$use_quartz" = "Yes"])
 
 dnl -----------------------------------
